src/web_socket_adapter.ts

```diff
@@ -0,0 +1,28 @@
+export interface WebSocketAdapter {
```

Comment:
nit: Please add doc comments to this file as well as the LICENSE header.

Reply:
178351b36f1b4d9a02094da6f454f77707b7a824